src/styles/_catalog.scss

```diff
@@ -1,22 +1,27 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  gap: 48px;
+  grid-template-columns: repeat(4, $card-width);
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (max-width: 1023px) {
+    grid-template-columns: repeat(3, $card-width);
+  }
+
+  @media (max-width: 767px) {
+    grid-template-columns: repeat(2, $card-width);
+  }
+
+  @media (max-width: 487px) {
+    grid-template-columns: $card-width;
+  }
```

Comment:
Would be nice to use mixins for media queries (usually we use them for media queries)
Or use loop here